### ISSUE TITLE
increase PoS reward

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#23]
goal: increase staking participation

questions:
* side effects on GRC value ?

mentioned in [hangout 22](https://github.com/Erkan-Yilmaz/Gridcoin-hangout-minutes/blob/master/hangout_2017_01_21.MD):

**Comment by grctest** [comment]
The GRC value wouldn't decrease significantly if the interest rate rose from 1.5 to say 3%, it would certainly be affected by a double/triple digit interest rate.

Perhaps a small increase in interest rate would increase staking participation.

**Comment by frank0051** [comment]
From a self-interest perspective I don't support because of this: https://cryptocointalk.com/topic/52084-question-about-interest-payment-frequency/

Haha, but in all seriousness, I would support a 3 month trial run to see if participation increases significantly, but we would need to get like a month's worth of baseline data first.

**Comment by skcin** [comment]
I don't think increasing the PoS reward will have any effect. Why would anyone stake more often you can still collect your interest at any time. The only thing this will change is the coin supply.

Besides a fixed stake reward there is another thing that could help and that is cold staking. 
If we could implement cold staking it would be much easier to keep staking 24/7. The best practise is to not stake on a full node because of security issues. What if we could eliminate these security issues.

The https://www.nem.io/ crypto currency has a even better system where you can stake without risk on unsecure systems. They call it delegated harvesting and it allows you to stake on a remote server without your wallet leaving your local pc. 

P.S. I create an issue for this

**Comment by grctest** [comment]
> Why would anyone stake more often you can still collect your interest at any time.

As reminded by @janko33bd in https://github.com/gridcoin/Gridcoin-Research/issues/106#issuecomment-274897104, POS2.0 removed coinage from the POS mechanism (offline interest building up is not a thing). Coinage is however still used in the DPOR mechanism (staking ability based on BOINC contributions increases over time even when client is offline).

So yeah, since offline interest is not being built up, the gridcoinstats.eu page ([example cpid with large unstaked balance](http://gridcoinstats.eu/cpid.php?a=view&id=e6a4984e4c9bb7b53ac56e4b03f0cd4f)) should actually read as 'This CPID has failed to claim an estimated 934 533,54 GRC of interest' rather than "There is also an estimate of 934 533,54 GRC unclaimed for interest."

Fixed stake rewards would mean that instead of earning like 4k in a single staked block with millions of GRC, they'd earn like 15.7 GRC (in order to get that 4k they'd need to stake 24/7).

**Comment by skcin** [comment]
> As reminded by @janko33bd in gridcoin/Gridcoin-Research#106 (comment), POS2.0 removed coinage from the POS mechanism (offline interest building up is not a thing).

I think that is wrong. coinage has been removed from the staking/probability part. The reward part still uses coinage to make sure you get the yearly interest. I think it is important that we do not mix up the security (probability to stake) issue with the reward issue. 

> So yeah, since offline interest is not being built up, the gridcoinstats.eu page (example cpid with large unstaked balance) should actually read as 'This CPID has failed to claim an estimated 934 533,54 GRC of interest' rather than "There is also an estimate of 934 533,54 GRC unclaimed for interest."

My understanding is that interest does build up. If these individuals open their wallets and stake they will be rewarded with the total interest they are owed. The only way to "destory" these interest is if you sent the coins to another adress before they stake.

**Comment by grctest** [comment]
Very good point, @skcin, you're correct that coinage is still taken into account for owed interest. To which, no, an increase in interest would likely change very little as users who keep their coins offline (building up interest) would earn the new interest rates regardless of their participation.

Removal of coinage from interest reward + fixed pos block rewards would be required before an increase in interest rates had a serious impact. That said, an increase in interest rates could attract new users?

**Comment by janko33bd** [comment]
Yes you are right, I've implemented staking only from POS 3.0, didn't look at rewards later.

**Comment by frank0051** [comment]
Bumping top given Fed Reserve interest rate increase. Treasuries are getting more competitive and that's going to continue to keep the dollar strong. The 12 month is now standing at .96% and the 6-month is at .85%. If we compare to currencies that have dollar pegs (mainly Middle East countries) the discount rates are creeping into the 2% or higher area.  If the Fed raises rates two more times this year - as forecast - GRCs get less competitive from a interest perspective.

Just sayin'...

**Comment by frank0051** [comment]
Just to memoralize an idea I put out there on: https://github.com/gridcoin/Gridcoin-Research/issues/106#issuecomment-300878994

- I am on record as supporting increasing the interest payments as the Federal Reserve hikes interest rates
- Perhaps, if there is interest (pun intended), a tiered interest system could be developed. Perhaps there is a base interest rate paid, but if you are on the network you can a "bonus" interest rate for helping secure the network.

**Comment by hovai** [comment]
If GRC begins to be viewed as a value holder, it loses all value.  The beauty of GRC is using computing power for real world progress.  I think the focus should remain on the development of GRC instead of its value.  GRC is one of the coins still around from 2014 because it never because a value oriented coin -- it was focused on BOINC and the dev team kept it focus toward improving those features.  

Why do you want to raise the interest gained from staking?

The way I see it, if the blockchain is developed, if BOINC is developed, and if GRC UI is developed, stakers will come.  Sure, you can encourage stake adoption through monetary incentives, but in the end is that what is wanted?  I prefer community outreach and marketing to encourage adoption.  Everyone I explain GRC/BOINC to, if they are at all technically inclined, sets up a wallet and BOINCS at night.  It is a slower route to further stake security, but it is true stake security -- people are investing their time into GRC, into understanding it and respecting it.  If people set up wallets for the higher interest, they might be a node for as long as GRC doesn't raise in price.  Once they see their exit point, they'll sell and move to the next coin.  So do we want to rush a change that focuses on the value of the coin instead of its function solely because crypto is getting a lot of attention right now?  The [GRC roadmap](https://www.tiki-toki.com/timeline/entry/759095/Gridcoin-Roadmap/) has a timeline out past 2018.  Once we have a secure stake base built on active users instead of investors, I would agree with perhaps maybe talking about raising the interest rate.  There are other ways to get investors to fund GRC development.


In regards to what's been said so far: 

There are some questions that need to be agreed upon:

Who do we want to reward more, stakers or BOINCers?  I like boincers
Who do we want to attract to GRC, investors or participators?  I like participators
Do we care what the price of GRC is today?  No.

- I believe that variable rate is, sorry, a terrible idea.  Particularly if it is tied to the fed and particularly at this time.  Crypto has roots in the idea of not having a fed arbitrarily raising rates.  Base it on math.

- I believe that increased visibility and increased education will provide increased stake security and that 
1.5% interest is fine, but whatever we decide upon should be fixed and if it is to be changed it should be a big deal (very big) and require an inclusive process and/or voting

- I believe that if you don't have your wallet staking, you shouldn't receive interest.  If you are not actively contributing, why do you get paid?

- The idea of a tiered interest rate structure is interesting: the longer you are on the network the higher the interest paid to you, or something.

**Comment by tomasbrod** [comment]
If it is really necessary to increase interest rate, we can pay that extra interest as fixed block reward.
Moving coin*age based interest to fixed block reward was already suggested, but it was dismissed as it would bread the initial agreement of paying 1.5%. By combining fixed pos block reward with coin*age interest we can solve two issues without breaking the agreement.

**Comment by denravonska** [comment]
@tomasbrod That is a really great idea!

**Comment by tomasbrod** [comment]
On the other hand I agree with @hovai in [his comment](https://github.com/gridcoin/Gridcoin-Research/issues/106#issuecomment-309146458) and I am against increasing interest rate.
And I prefer moving some % of interest from coin*age reward to fixed block reward.
But please, take above as my view, Rob @gridcoin is the one to decide on the interest changes.

**Comment by 3ullShark** [comment]
I think we should drop interest all together. Inflation will get out of control of its kept the way it is.

**Comment by frank0051** [comment]
> I think we should drop interest all together. Inflation will get out of control of its kept the way it is.

ROFL! A fixed interest rate at 1.5% leads to inflation getting out of control...said no one ever....

**Comment by 3ullShark** [comment]
1.5% might not seem much now but the larger the coin supply gets the bigger it will get. Or am I missing something?

**Comment by frank0051** [comment]
> 1.5% might not seem much now but the larger the coin supply gets the bigger it will get. Or am I missing something?

Its still just a constant rate that allows for growth in the money supply - period. Generally speaking, expansion of the money supply - particularly at this slow level - isn't tied to inflation. It's not exponential growth in the money supply, so it cannot get, "out of control" in the way that you're [probably unintentionally] implying. In fact, scarcity is more likely to lead to inflation (or, alternatively, the flat out dumping of the coin for another alt coin) than a steady-state increase in the money supply.

I think the earlier conversations on this issue were right: we still need something that provides incentives for the investors to stake because right now they hold an immense amount of the coinage. This for me isn't an issue of inflation (which 1.5% interest won't cause anyways - some Arab countries have interest rates in excess of 4-5% and have 2% inflation or even deflation); it also isn't a big issue of supporting BOINCing (continuing to offer POR will do so); rather, the issue at hand here is network security. I don't want to go back to do the days when the network weight is 30M when the coin has some serious value - that can lead to some scary things...
